modules/mysql/examples_test.go

```diff
@@ -66,7 +66,10 @@ func ExampleRun_connect() {
 		}
 	}()
 
-	connectionString, _ := mysqlContainer.ConnectionString(ctx)
+	connectionString, err := mysqlContainer.ConnectionString(ctx)
+	if err != nil {
+		log.Fatalf("failed to get connection string: %s", err)
```

Comment:
```suggestion
		log.Fatalf("failed to get connection string: %s", err) // nolint:gocritic
```

Reply:
FYI: This gets reworked in a latter PR as all the examples are broken

Reply:
Broken in what manner? Is the lint complaining?

Reply:
If an error occurs clean up doesn't happen as `log.Fatalf` issues an `os.Exit(1)` so the defers don't run.

Reply:
Ah oh yes, sure, thanks for the explanation. You're very right 🙇